app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's a good practice to annotate the types of elements inside the list. In this case, it should be `list[Person]` if `Person` is a class of friends.